Cubical/Data/Int/Properties.agda

```diff
@@ -233,6 +205,55 @@ ind-assoc _·_ f g p q base m n (suc o) =
     where
     IH = ind-assoc _·_ f g p q base m n o
 
++-comm : ∀ m n → m + n ≡ n + m
++-comm m (pos n) = ind-comm _+_ pos sucInt refl sucInt+ +sucInt pos0+ m n
++-comm m (negsuc n) = ind-comm _+_ negsuc predInt refl predInt+ +predInt negsuc0+ m n
+
 +-assoc : ∀ m n o → m + (n + o) ≡ (m + n) + o
 +-assoc m n (pos o) = ind-assoc _+_ pos sucInt +sucInt refl (λ _ _ → refl) m n o
 +-assoc m n (negsuc o) = ind-assoc _+_ negsuc predInt +predInt refl +predInt m n o
+
+-- Alternate definition: Compose sucPathInt with itself n times. Transporting along this
+-- will be addition, transporting with it backwards will be subtraction.
+
+addEq subEq : ℕ → Int ≡ Int
+addEq zero = refl
+addEq (suc n) = compPath (addEq n) sucPathInt
+
+subEq n i = addEq n (~ i)
+
+_+'_ : Int → Int → Int
+m +' pos n    = transport (addEq n) m
+m +' negsuc n = transport (subEq (suc n)) m
+
+private
+  FamilyOfEquiv : (f : Int → Int → Int) → Set
```

Comment:
This definition is never used again? Why keep it?

Reply:
Oh wait, it was used again later... How about moving it into a `where`-clause in that definition instead?